torchvision/csrc/io/image/cpu/decode_png.cpp

```diff
@@ -71,6 +71,11 @@ torch::Tensor decode_png(const torch::Tensor& data, ImageReadMode mode) {
     TORCH_CHECK(retval == 1, "Could read image metadata from content.")
   }
 
+  if (bit_depth != 8) {
```

Comment:
Should this be > 8?
looks like we have tests for 1-bit png that are failing now https://app.circleci.com/pipelines/github/pytorch/vision/9243/workflows/a1127581-3c33-48db-b2fb-f619e99e565d/jobs/677291/tests#failed-test-0

Reply:
Right, I am at supermarket, wait…